### PR TITLE
Add test case for iot-simplified-installer

### PR DESCRIPTION
**Pull request by yih-redhat**
This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

**Comment by yih-redhat** [comment]
The latest osbuild-composer on fedora does not have iot-simplified-installer and iot-qcow2-image. But I can build rpm by myself and then I can see them.

@paulwhalen  Tested iot-simplified-installer image, still have these two issues:
https://github.com/osbuild/osbuild-composer/issues/3533
https://github.com/osbuild/osbuild-composer/issues/3529

**Comment by yih-redhat** [comment]
Tested iot-simplified-installer on fedora 38 and fedora 39, fdo onboarding disabled, firstboot and embeded ignition enabled, works as expected, except known issues.

**Review by achilleas-k — APPROVED** [review]
Thanks!

**Comment by achilleas-k** [comment]
There was something strange with the commits in this PR.  The second commit was a merge commit and it couldn't be rebased, so I had to amend it.  Should be fine now.

**Review by achilleas-k — APPROVED** [review]

File: test/cases/ostree-simplified-installer.sh [hunk 1]
```diff
@@ -124,6 +124,7 @@ KERNEL_RT_PKG="kernel-rt"
 SYSROOT_RO="false"
 ANSIBLE_USER="admin"
 FDO_USER_ONBOARDING="false"
+IMAGE_TYPE=redhat
```
Comment (henrywang) [inline test/cases/ostree-simplified-installer.sh:127]: What does this `IMAGE_TYPE` mean?

File: test/cases/ostree-simplified-installer.sh [hunk 2]
```diff
@@ -173,8 +186,15 @@ if [[ "$FDO_USER_ONBOARDING" == "true" ]]; then
 fdouser ALL=(ALL) NOPASSWD: ALL
 EOF
     sudo /usr/local/bin/yq -iy '.service_info.files |= [{path: "/etc/sudoers.d/fdouser", source_path: "/tmp/fdouser"}]' /etc/fdo/aio/configs/serviceinfo_api_server.yml
-    sudo systemctl restart fdo-aio
 fi
+
+# Fedora iot-simplified-installer uses /dev/vda3, https://github.com/osbuild/osbuild-composer/issues/3527
```
Comment (henrywang) [inline test/cases/ostree-simplified-installer.sh:191]: Could you please move this part into line 28?

**Review by henrywang — APPROVED** [review]

**Review by henrywang — CHANGES_REQUESTED** [review]
@yih-redhat I have my comment inline. Thanks.

File: test/cases/ostree-simplified-installer.sh (hunk 2 above)
Comment (yih-redhat) [reply test/cases/ostree-simplified-installer.sh]: Thanks, fixed.

**Comment by yih-redhat** [comment]
Blocked by bug https://github.com/osbuild/osbuild-composer/issues/3726

**Comment by croissanne** [comment]
Rebased to prevent OCI private key leaking. Apologies :(

**Comment by yih-redhat** [comment]
Need to update snapshot first to use new selinux packages in f39 repo which includes the fix for bug https://github.com/osbuild/osbuild-composer/issues/3726

**Pull request closed without merging**